application/src/main/java/org/togetherjava/tjbot/commands/system/BotCore.java

```diff
@@ -290,6 +308,40 @@ public void onUserContextInteraction(final UserContextInteractionEvent event) {
                 UserContextCommand.class).onUserContext(event));
     }
 
+    /**
+     * Returns the {@link ComponentId} of the given componentId. If the component has expired, the
+     * user gets a message stating that.
+     *
+     * @param event the {@link IReplyCallback event} to reply to
+     * @param componentId the component's ID
+     *
+     * @return the relating {@link ComponentId}
+     */
+    @Nullable
+    public ComponentId handleGetComponentId(IReplyCallback event, String componentId) {
+        Optional<ComponentId> componentIdOpt;
+        try {
```

Comment:
this javadoc needs more details. right now it says "gets the foo of the given foo". which makes no sense.

what it actually does is:
1. parse the component id (string -> domain object)
2. handle any errors

so i would rename it to `handleParseComponentId` and adjust the javadoc accordingly, i.e. "parses the give component id and handles error cases. for example if the component has expired, the user gets a warning message"

Reply:
i dont like the name and the javadoc.
* rename to `handleParseComponentId`
* change javadoc description to `Parses the given component ID and handles error cases. For example if the component has expired, the user gets a warning message.`